src/screens/reader/components/ReaderBottomSheet/ReaderValueChange.tsx

```diff
@@ -3,43 +3,61 @@ import React from 'react';
 
 import { useChapterReaderSettings, useTheme } from '@hooks/persisted';
 import { IconButtonV2 } from '@components';
-import { getString } from '@strings/translations';
 
-interface ReaderLineHeightProps {
+type ValueKey<T extends object> = {
+  [K in keyof T]: T[K] extends number ? K : never;
+}[keyof T];
+
+interface ReaderValueChangeProps {
   labelStyle?: TextStyle | TextStyle[];
+  valueChange?: number;
+  label: string;
+  valueKey: ValueKey<ReturnType<typeof useChapterReaderSettings>>;
+  min?: number;
+  max?: number;
 }
 
-const ReaderLineHeight: React.FC<ReaderLineHeightProps> = ({ labelStyle }) => {
+const ReaderValueChange: React.FC<ReaderValueChangeProps> = ({
+  labelStyle,
+  label,
+  valueChange = 0.1,
+  valueKey,
+  min = 1.3,
+  max = 2,
+}) => {
   const theme = useTheme();
-
-  const { lineHeight, setChapterReaderSettings } = useChapterReaderSettings();
+  const { setChapterReaderSettings, ...settings } = useChapterReaderSettings();
 
   return (
     <View style={styles.container}>
       <Text style={[{ color: theme.onSurfaceVariant }, labelStyle]}>
-        {getString('readerScreen.bottomSheet.lineHeight')}
+        {label}
       </Text>
       <View style={styles.buttonContainer}>
         <IconButtonV2
           name="minus"
           color={theme.primary}
           size={26}
-          disabled={lineHeight <= 1.3}
+          disabled={settings[valueKey] <= min}
           onPress={() =>
-            setChapterReaderSettings({ lineHeight: lineHeight - 0.1 })
+            setChapterReaderSettings({
+              [valueKey]: settings[valueKey] - valueChange,
```

Comment:
```suggestion
              [valueKey]: Math.max(min, settings[valueKey] - valueChange),
```

Reply:
I mean we can do it like this, but the button already gets disabled ```ts disabled={settings[valueKey] <= min} ```